src/App.tsx

```diff
@@ -31,7 +31,12 @@ function App() {
           {/* Page de recherche par categorie via navbar */}
           <Route path="/catalog/:categoryId" element={<Catalog />} />
 
-
+          {/* Page de settings du profil */}
+          <Route path="/settings/:section" element={<Settings />} />
+          <Route
+            path="/settings"
+            element={<Navigate to="/settings/profile" />}
```

Comment:
Navigate dans un element ???

Reply:
J'ai pas beaucoup d'expérience avec react-router-dom donc je ne sais pas quelles sont les bonnes pratiques.

Qu'est-ce que tu proposes ?

Reply:
Ca depend ce que tu veux afficher et surtout depuis où
Perso j'avais mis ca:  
{/*Page de profil de l'utilisateur*/}
<Route path="/member/:userId" element={<User />} />

Je vois pas trop ce que tu veux faire avec 
<Route path="/settings/:section" element={<Settings />} />
          <Route
            path="/settings"
            element={<Navigate to="/settings/profile" />}
Si c'est pour afficher la page de profil ET la page de modification de profil c'est pas la peine, en faite depuis la page de profil en cliquant sur Modifier je réouvre le formulaire en ecriture donc c'est la meme page

Reply:
Le but est de faire une redirection automatique : https://www.vinted.fr/settings redirige automatiquement vers https://www.vinted.fr/settings/profile